### DISCUSSION TITLE
get_drawings()  is no coordinate show width and height

### DISCUSSION THREAD

**Discussion by zdw1011781461** [pymupdf/PyMuPDF#1065] [Q&A] [answered]
I have a need to convert the curve to SVG, I use svg package "cairocffi"
cairo.SVGSurface('./svg/svg%s.svg' % index,  "width", "height")  get_drawings() is not exist "width", "height"

**Comment by JorjMcKie** [comment, marked as answer]
I suppose you have figured out how to locate curves and other stuff in the output of `page.get_drawings()`.
1. `get_drawings()` is a list of Python dicts.
2. Each dict (called a "path") is a list of elementary drawings (lines, curves, rectangles) accompanied by a number of shared properties.
3. Among those path properties is `"rect"` which is the rectangle wrapping the complete path.
4. To compute the rectangle of a single curve within a path, simply do this:

```python
# a curve is a tuple ("c", p1, p2, p3, p4). The items after the "c" are points
r = fitz.Rect(p1, p1)
r |= p2
r |= p3
r |= p4
# now 'r' is the rectangle wrapping that curve. Take its width / height
```
